test/cases/wasm/decoding/index.js

```diff
@@ -1,13 +1,13 @@
 it("should support wasm compiled from c++", function() {
-	return import("./memory3.wasm").then(function(wasm) {
+	return import("./memory3.wast").then(function(wasm) {
```

Comment:
I would keep the `decoding` tests in `wasm` as they try to test special encodings of the wasm binary (like paddings). This is not possible with `wast` (`wat`)

Reply:
The other tests can be changed to `.wat` for better readability.

Reply:
Good point yes